app/models/role.rb

```diff
@@ -566,6 +587,20 @@ def self.permissions_with_descriptions
           'Cohorts',
         ],
       },
+      can_view_deleted_cohort_clients: {
```

Comment:
I think the implementation of this tab got missed.

Reply:
You are correct! 😄